ssh-pkcs11.c

```diff
@@ -77,6 +77,15 @@ struct pkcs11_key {
 	int			keyid_len;
 };
 
+#ifdef WINDOWS
```

Comment:
I don't think the changes in this file are required as we have a copy of these functions/structs in ssh-pkcs11-client.c

Reply:
Yup, you're right. Removed the unnecessary parts here.